tests/utils.py

```diff
@@ -1026,7 +1027,7 @@ def snapshot_context(
             while r is None and time.time() - attempt_start < 60:
                 try:
                     r = conn.getresponse()
-                except http.client.RemoteDisconnected:
```

Comment:
ddtrace.ext.http is shadowing the built http module. This results in this error: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/64068/workflows/51694040-d0a5-4416-ba41-37f25e3b431a/jobs/3968306